src/wallet/mod.rs

```diff
@@ -4728,7 +4771,7 @@ pub(crate) mod test {
         // ...Now the coinbase is mature :)
         let mut builder = wallet.build_tx();
         builder
-            .add_recipient(addr.script_pubkey(), balance / 2)
+            .add_recipient(addr.script_pubkey(), balance.immature / 2)
             .set_current_height(maturity_time);
         builder.finish().unwrap();
```

Comment:
Check that the available amount is non-zero, and that the immature amount is zero

Reply:
This ^

Reply:
@wszdexdrf why has this been resolved? Is it fixed? It doesn't seem to me...